node/src/types/block.rs

```diff
@@ -1250,7 +1248,7 @@ pub(crate) mod json_compatibility {
                 hash: block.hash,
                 header: JsonBlockHeader::from(block.header),
                 body: block.body,
-                proofs: block.proofs.into_iter().map(JsonProof::from).collect(),
+                proofs: vec![],
```

Comment:
I'd be in favour of removing this `From` impl since we can't _really_ construct a `JsonBlock` properly from just a `Block` any more.  The `JsonBlock` represents a block and its signatures, so I feel it's a little dangerous to have a public constructor of `JsonBlock` which doesn't properly construct it.

I think we should either rely on your `new()` constructor, or we could do a `From<(Block, BlockSignatures)>` if you really want.